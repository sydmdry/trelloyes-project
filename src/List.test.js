import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import List from './List';

describe('List component', () => {
  it('List renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<List 
    key='0'
    header='test'
    cards={[]}
    />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('List renders the UI as expected', () => {
    const tree = renderer
      .create(<List key='0' header='test-2' cards={[]}/>)
      .toJSON();
    expect(tree).toMatchSnapshot();  
  });
});