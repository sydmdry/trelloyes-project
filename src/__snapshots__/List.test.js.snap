// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`List component List renders the UI as expected 1`] = `
<section
  className="List"
>
  <header
    className="List-header"
  >
    <h2>
      test-2
    </h2>
  </header>
  <div
    className="List-cards"
  >
    <button
      className="List-add-button"
      type="button"
    >
      + Add Random Card
    </button>
  </div>
</section>
`;
